.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
FWIW I typically put these in a global ignore file since it's IDE-dependent.

Reply:
niiiice! didn't know about this one! ty!